jmclient/jmclient/configure.py

```diff
@@ -569,10 +555,20 @@ def get_blockchain_interface_instance(_config):
             rpc_wallet_file)
         if source == 'bitcoin-rpc': #pragma: no cover
             bc_interface = BitcoinCoreInterface(rpc, network)
+            if testnet:
+                btc.select_chain_params("bitcoin/testnet")
```

Comment:
Another reminder: `CBitcoinKey`, `P2WPKHBitcoinAddress` etc will still use mainnet prefixes in bitcointx. To be chain-generic, use `CCoinKey`, `P2WPKHCoinAddress` etc.

Reply:
Addressed above, thanks ( https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/1e2fc3848295a05f4ee685fd435f277c378acadc )